app/services/support_interface/ministerial_report_export.rb

```diff
@@ -0,0 +1,52 @@
+module SupportInterface
+  class MinisterialReportExport
+    def call
+      results = subject_mapping(subject_status_count)
+
+      rows = {}
+
+      Subject::SUBJECTS.each do |subject|
+        rows[subject] = {
+          applications: 0,
+          offer_received: 0,
+          accepted: 0,
+          application_declined: 0,
+          application_rejected: 0,
+          application_withdrawn: 0,
+        }
+      end
+
+      results.each do |key, count|
```

Comment:
this probably needs to be batched because there's going to be a loooooot of operations going on in here 

```
 results.each do |key, count|

```

Reply:
I don't think `find_each` works here as results is now a Hash rather than ActiveRecord?